### DISCUSSION TITLE
Any way to disable notifications for a bulk issue change via the API?

### DISCUSSION THREAD

**Discussion by roji** [community/community#36305] [Apps, API and Webhooks]
We're planning a massive bulk change to e.g. all our repo issues (via the REST or GraphQL APIS), but we'd like to prevent a notification from being generated for each and every issue. Is there any way to disable notifications generated specifically from an API-triggered change, or similar?

**Comment by ossanna16** [comment]
Hey there @roji and thank you for asking a good question! I'm glad you found our community 🙂
